src/parquet/column/properties.h

```diff
@@ -81,8 +82,11 @@ static int64_t DEFAULT_PAGE_SIZE = 1024 * 1024;
 static int64_t DEFAULT_DICTIONARY_PAGE_SIZE = DEFAULT_PAGE_SIZE;
 static bool DEFAULT_IS_DICTIONARY_ENABLED = true;
 static Encoding::type DEFAULT_ENCODING = Encoding::PLAIN;
+static Encoding::type DEFAULT_DICTIONARY_ENCODING = Encoding::PLAIN_DICTIONARY;
```

Comment:
From reading the `parquet-mr` code I got at least the impression that the only difference are the used enums, i.e. at least the output of dictionary-encoded ColumnChunks is identical in v1 and v2 and only the declared types are different.